contracts/src/v0.8/dev/ArbitrumValidator.sol

```diff
@@ -240,41 +286,48 @@ contract ArbitrumValidator is TypeAndVersionInterface, AggregatorValidatorInterf
       return true;
     }
 
-    int isServiceOffline = 1;
-    // NOTICE: if gasCostL2 is zero the payment is processed on L2 so the L2 address needs to be funded, as it will
+    // Encode the Forwarder call
+    bytes4 selector = ForwarderInterface.forward.selector;
+    address target = L2_FLAGS;
+    // Choose and encode the underlying Flags call
+    bytes memory data = currentAnswer == ANSWER_SEQ_OFFLINE ? CALL_RAISE_FLAG : CALL_LOWER_FLAG;
+    bytes memory message = abi.encodeWithSelector(selector, target, data);
+    // Make the xDomain call
+    // NOTICE: if gasCostL2Value is zero the payment is processed on L2 so the L2 address needs to be funded, as it will
     // paying the fee. We also ignore the returned msg number, that can be queried via the InboxMessageDelivered event.
-    s_inbox.createRetryableTicket{value: s_gasConfig.gasCostL2}(
-      s_l2FlagsAddress,
+    IInbox(CROSS_DOMAIN_MESSENGER).createRetryableTicketNoRefundAliasRewrite{value: s_gasConfig.gasCostL2Value}(
```

Comment:
`createRetryableTicketNoRefundAliasRewrite` I imagine doesn't change `refundAddr` address to the new L2 alias right?

Reply:
Yes, exactly! The old `createRetryableTicket` also does alias the `refundAddr` addr, so we use this one instead to keep it straightforward.